pydecred/dcrdata.py

```diff
@@ -1626,3 +1635,38 @@ def purchaseTickets(self, keysource, utxosource, req):
                 )
             )
         return (splitTx, tickets), splitSpent, internalOutputs
+
+    def revokeTicket(self, tx, keysource, redeemScript):
+        """
+        Revoke a ticket by signing the supplied redeem script and broadcasting the raw transaction.
+
+        Args:
+            tx (object): the msgTx of the ticket purchase.
+            keysource (object): a KeySource object that holds a function to get the private key used for signing.
+            redeemScript (byte-like): the 1-of-2 multisig script that delegates voting rights for the ticket.
```

Comment:
I recently realized that it's actually `bytes-like`, not `byte-like`. I've used it wrong all over the place though.

Reply:
Let's do this with a sed sometime.